diagnostic.md

```diff
@@ -183,6 +193,4 @@ with the class `.stretch` be at a _screen width_ of 550px?
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+Width of Element in .Stretch will be 25px in that screen;
```

Comment:
`137.5 px`. This is because at `@media (min-width= 400px)` the `.stretch` class will have a width of 25%. So at 550px, 25% is 137.5px.